src/JsonSchema/Uri/UriRetriever.php

```diff
@@ -44,6 +52,16 @@ class UriRetriever implements BaseUriRetrieverInterface
      */
     private $schemaCache = array();
 
+    /**
+     * Adds an endpoint to the media type validation blacklist
+     *
+     * @param string $endpoint
+     */
+    public function addBlacklistedEndpoint($endpoint)
```

Comment:
Again with the naming. I don't feel that this is a blacklist - it's a list of exceptions.

Reply:
@estahn ☝️